client/my-sites/checkout/get-thank-you-page-url/index.ts

```diff
@@ -542,20 +549,18 @@ function getFallbackDestination( {
 
 	const marketplaceProducts =
 		cart?.products?.filter( ( product ) => product?.extra?.is_marketplace_product ) || [];
-
-	const marketplacePluginSlugs = marketplaceProducts
-		.filter(
-			( { extra } ) =>
-				extra.product_type === 'marketplace_plugin' || extra.product_type === 'saas_plugin'
-		)
-		.map( ( { extra } ) => extra.product_slug );
-
-	const marketplaceThemeSlugs = marketplaceProducts
-		.filter( ( { extra } ) => extra.product_type === 'marketplace_theme' )
-		.map( ( { extra } ) => extra.product_slug );
-
```

Comment:
Minor  cleanup (We do not need to resolve this variables if not a marketplace product.

CC - @gcsecsey